domains/varun-gotmare.json

```
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "VarunGotmare"
+  },
+  "records": {
+    "A": [
+      "216.198.79.1"
+    ],
+    "TXT": {
+      "_vercel": "vc-domain-verify=varun.is-a.dev,a5471e7e697265ce688a"
+    }
```

This is not how you do it!
create another json file "_vercel.varun-gotmare.json" and add the TXT Records there

for more info, read the guide -> https://docs.is-a.dev/guides/vercel/


```suggestion
    ]
```